src/Client/Updater.php

```diff
@@ -28,21 +28,27 @@ class Updater
    */
     protected $mirrors;
 
-    protected $keyRegistry;
```

Comment:
$keyRegistry was never used.

Reply:
I think this comment is just an FYI on why the change was made rather than a request for feedback, so marking resolved. Thanks!